### ISSUE TITLE
Communicate with Kaizen users that Kaizen will be shut down in 2 weeks

### ISSUE THREAD

**Issue by joachimweyl** [CCI-MOC/ops-issues#816]
- [x] Draft email
- [x] 2+ Team members review email
- [x] Implement suggested updates
- [x] Wait for the [OSticket issue](https://github.com/cci-moc/ops-issues/issues/748) to be resolved
- [x] Update dates based on when we can send the email
- [x] Send email
- [x] when we send the email check the next day for bounced emails from openshift users.

**Comment by joachimweyl** [comment]
email draft:
Subject: March 6th MOC Kaizen OpenStack & OpenShift Cluster Final Shutdown
To be sent: Feb 20, 2023 (change dates in the email if not sent by this date)

Hello Kaizen OpenStack & OpenShift 3.x Users,
We launched the Mass Open Cloud (MOC) with a vision of creating a production cloud that would enable innovation by a broad industry and research community.
The MOC has successfully supported hundreds of projects supporting thousands of users, transitioned research projects into upstream software artifacts, and formed a community of interrelated research projects.
One of those projects is the New England Research Cloud (NERC), a production cloud with professional operations and a team of experienced facilitators. The NERC supports OpenStack users today and will soon support OpenShift users. With that in mind, on March 6th MOC Kaizen OpenStack & OpenShift 3.x clusters will be shut down and repurposed.
The window for OpenStack volume migration is over. If you were unable to migrate to the NERC OpenStack by this time please make sure to create backups of your volumes to store locally.
The older OpenShift 3.x cluster will also be shut down and if you wish to backup any of your information please do so. This will not affect OpenShift 4.x clusters.
If you have any additional questions, please create a ticket [here](https://osticket.massopen.cloud/login.php).
Thanks!
Michael Daitzman
PS – If you were forwarded this email from a colleague and would prefer to receive these notifications directly, you may sign up for the kaizen-usersmailing list by sending an email to: kaizen-users-join@lists.massopen.cloud

**Comment by joachimweyl** [comment]
@msdisme, @naved001, @rob-baron, @knikolla, @larsks, and @hakasapl would 2-3 of you please review the communication above?

**Comment by naved001** [comment]
> The older OpenShift 3.x cluster will also be shut down and if you wish to backup any of your information please do so. 

How would one do it if it's already powered off? Do we expect to power it back on?

**Comment by larsks** [comment]
> The older OpenShift 3.x cluster will also be shut down and if you wish to backup any of your information please do so. This will not affect OpenShift 4.x clusters.

I would rephrase this to something along the lines of, "if you wish to backup any of your information from this environment please open a ticket at ... and we will assist you".

As Naved mentioned, this environment has *already* been shut down, so it may require some work to re-invigorate it to the point that we can retrieve data.

**Comment by joachimweyl** [comment]
If somebody reaches out to state that they have important information they MUST retrieve (seems unlikely based on no response to the power off) then we would want to look into what is required to turn it back on.

**Comment by joachimweyl** [comment]
> I would rephrase this to something along the lines of, "if you wish to backup any of your information from this environment please open a ticket at ... and we will assist you".
`done`

**Comment by joachimweyl** [comment]
I have updated the text for a new mailing date.

**Comment by joachimweyl** [comment]
on hold, removing from sprint.

**Comment by joachimweyl** [comment]
@msdisme updated the dates in the email above. Please send this on April 12th when you get back from your vacation.

**Comment by joachimweyl** [comment]
@msdisme did we get any bounce messages when we sent this message that we should look into?
